tools/dockerfile/interoptest/grpc_interop_cxx_xds/Dockerfile.xds_server

```diff
@@ -27,6 +27,13 @@ COPY . .
 RUN tools/bazel build //test/cpp/interop:xds_interop_server
 RUN cp -rL /workdir/bazel-bin/test/cpp/interop/xds_interop_server /artifacts/
 
+# Download grpcurl
+RUN curl -L https://github.com/fullstorydev/grpcurl/releases/download/v1.8.7/grpcurl_1.8.7_linux_x86_64.tar.gz -o grpcurl.tar.gz && \
```

Comment:
Not too wild about having to embed grpcurl in the image. This will require making this change in all lang-specific images.
I'm not opposing this, especially short-term. But we should discuss alternatives.

Reply:
There are only two types of pre-stop hook: exec and HTTP. There's no gRPC pre-stop hook. I don't see any alternative here.

Reply:
Yea, I've confirmed this too. However, `exec` still provides alternatives.